### PR TITLE
chore: add another solution for help

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

### CODE REVIEW

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/input/examples/10/index.html [hunk 1]
```diff
@@ -54,5 +54,10 @@ <h4>
     <code>autocomplete="new-password"</code>
     We could use
     <code>new-password</code>
-    , which should tell the browser, that this is like a new password and autofill is therefore not useful here.
+    which should tell the browser, that this is like a new password and autofill is therefore not useful here.
+</p>
+
+<p>
+    Another way you can use this combination of html attributes:
+    <code>&lt;input role="presentation" autocomplete="off" /&gt;</code>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input/examples/10/index.html:62]: nit:

Provide source-link where our users can read more about it, please

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/input/examples/10/index.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/input/examples/10/index.html:62]: Is it okey?
https://gist.github.com/niksumeiko/360164708c3b326bd1c8?permalink_comment_id=3793032

I can't find official solution, but I check `<input role="presentation" autocomplete="off" />` it's works

Comment (nsbarsukov) [reply projects/demo/src/modules/components/input/examples/10/index.html:62]: This comment ?

https://gist.github.com/niksumeiko/360164708c3b326bd1c8?permalink_comment_id=3925346#gistcomment-3925346

Comment (nsbarsukov) [reply projects/demo/src/modules/components/input/examples/10/index.html:62]: Let add link
Someday, it stops working.
At least, our users can open this link to find new solution :D

**Pull request merged**